openassessment/templates/openassessmentblock/edit/oa_edit_staff_assessment.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+{% load tz %}
+{% spaceless %}
+<li class="openassessment_assessment_module_settings_editor" id="oa_staff_assessment_editor">
+    <div class="drag-handle action"></div>
+    <div class="openassessment_inclusion_wrapper">
+        <input type="checkbox" id="include_staff_assessment"
+                {% if assessments.staff_assessment.required %} checked="true" {% endif %}>
+        <label for="include_staff_assessment">{% trans "Step: Staff Assessment" %}</label>
```

Comment:
I believe the preferred syntax is to next the input within the label (correct, @cptvitamin?). This removes the necessity of using an ID to link the two together, and we want to avoid IDs if at all possible.

That being said, I'm sure you following the format of the other templates. So if you wish to leave this and we fix them all at once, I'm OK with that.